recommenders/utils/spark_utils.py

```diff
@@ -11,25 +10,28 @@
     pass  # skip this import if we are in pure python environment
 
 
+MMLSPARK_PACKAGE = "com.microsoft.ml.spark:mmlspark_2.11:0.18.1"
```

Comment:
Do we need these lines now?

Reply:
Yes, for the mmlspark_lightgbm_criteo.ipynb notebook and databricks_install.py